README.md

```diff
@@ -386,6 +385,54 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Method Ordering
+
+Method implementations are grouped and ordered with lifecycle methods first, then by class inheritance. A `pragma mark -` is used to separate each group.
```

Comment:
I really like the direction of this generally, but I’m *very concerned* about this first part.

Having lifecycle methods be "roughly defined" will I think lead to a lot of confusion down the road for developers who rely on this guide. This pull request also doesn’t lay out any benefits whatsoever to separating them from their natural place in the inheritance hierarchy.

I’d be happy to lay out my thoughts in more detail, but generally, I think that this is dangerous territory because it will make the guide even harder to follow in practice. It gives developers another question that they have to hold in their heads whenever they’re adding method implementations: "is this a _lifecycle_ method?".

I’d be on board with that additional complexity if I saw the tradeoff as having some worthwhile benefit, but it does not.